### PR TITLE
Add Backup Payment instruments

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

**Review by akozhevnikov-rebilly — DISMISSED** [review]

**Review by justnero — DISMISSED** [review]

**Review by ninjasimon — DISMISSED** [review]

File: openapi/components/schemas/Common/CommonAlternativeInstrument.yaml [hunk 1]
```diff
@@ -28,6 +28,10 @@ properties:
     enum:
       - active
       - deactivated
+  useAsBackup:
+    description: Allow to use this payment instrument as backup, it will be used on invoice payment retry.
```
Comment (adamaltman) [inline openapi/components/schemas/Common/CommonAlternativeInstrument.yaml:32]:
```suggestion
    description: Allow using this payment instrument as a backup for invoice payment retries.
```

File: openapi/components/schemas/Common/CommonBankAccount.yaml [hunk 2]
```diff
@@ -61,6 +61,10 @@ properties:
     enum:
       - active
       - deactivated
+  useAsBackup:
+    description: Allow to use this payment instrument as backup, it will be used on invoice payment retry.
```
Comment (adamaltman) [inline openapi/components/schemas/Common/CommonBankAccount.yaml:65]:
```suggestion
    description: Allow using this payment instrument as a backup for invoice payment retries.
```

File: openapi/components/schemas/Common/CommonPayPalAccount.yaml [hunk 3]
```diff
@@ -35,6 +35,10 @@ properties:
       - inactive
       - active
       - deactivated
+  useAsBackup:
+    description: Allow to use this payment instrument as backup, it will be used on invoice payment retry.
```
Comment (adamaltman) [inline openapi/components/schemas/Common/CommonPayPalAccount.yaml:39]:
```suggestion
    description: Allow using this payment instrument as a backup for invoice payment retries.
```

File: openapi/components/schemas/Common/CommonPaymentCard.yaml [hunk 4]
```diff
@@ -70,6 +70,10 @@ properties:
     description: The billing address.
     allOf:
       - $ref: ../Contact/ContactObject.yaml
+  useAsBackup:
+    description: Allow to use this payment instrument as backup, it will be used on invoice payment retry.
```
Comment (adamaltman) [inline openapi/components/schemas/Common/CommonPaymentCard.yaml:74]:
```suggestion
    description: Allow using this payment instrument as a backup for invoice payment retries.
```

**Review by adamaltman — CHANGES_REQUESTED** [review]
chore: minor edits

**Review by adamaltman — APPROVED** [review]

**Review by subenson — APPROVED** [review]

**Review by vykhrystiuk — APPROVED** [review]

**Pull request merged**